app/scripts/metamask-controller.js

```diff
@@ -3374,17 +3377,11 @@ export default class MetamaskController extends EventEmitter {
     this.setLedgerTransportPreference(transportPreference);
   }
 
-  async _loginUser() {
+  async _loginUser(password) {
```

Comment:
Do we want to add this argument to the function call as well?

Reply:
done with most recent cherry-pick